src-docs/src/views/tables/mobile/mobile.js

```diff
@@ -118,7 +118,7 @@ export const Table = () => {
             )
           : undefined,
         header: customHeader ? false : true,
-        fullWidth: customHeader ? true : false,
+        width: customHeader ? '100%' : 'auto',
```

Comment:
Let's do undefined here so that instead of **setting** it, it just nulls out the prop.

```suggestion
        width: customHeader ? '100%' : undefined,
```